aws-connect-tasktemplate/template.yml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: AWS SAM template for the AWS::Connect::TaskTemplate resource type
+
+Globals:
+  Function:
```

Comment:
Any reason why Timeout is not 180 and Memory is not 256 like other resources ?

Reply:
This depends.  Our resource could not launch with that configure, so I have to update it